tests/conftest.py

```diff
@@ -65,9 +77,50 @@ def datacite_record_all_fields():
     return Datacite("cool-repo", source_records)
 
 
+# dspace_dim ##########################
+
+
 @pytest.fixture
-def aardvark_records():
-    return JSONTransformer.parse_source_file("tests/fixtures/aardvark_records.jsonl")
+def dspace_dim_record_all_fields():
+    source_records = DspaceDim.parse_source_file(
+        "tests/fixtures/dspace/dspace_dim_record_all_fields.xml"
+    )
+    return next(source_records)
+
+
+@pytest.fixture
+def dspace_dim_record_attribute_and_subfield_variations():
+    source_records = DspaceDim.parse_source_file(
+        "tests/fixtures/dspace/dspace_dim_record_attribute_and_subfield_variations.xml"
+    )
+    return next(source_records)
+
+
+@pytest.fixture
+def dspace_dim_record_errors():
+    source_records = DspaceDim.parse_source_file(
+        "tests/fixtures/dspace/dspace_dim_record_errors.xml"
+    )
+    return next(source_records)
+
+
+@pytest.fixture
+def dspace_dim_record_optional_fields_blank():
+    source_records = DspaceDim.parse_source_file(
+        "tests/fixtures/dspace/dspace_dim_record_optional_fields_blank.xml"
```

Comment:
We should likely rename this fixture and `_optional_fields_missing` during the refactor